app/src/main/java/com/example/triptracker/view/map/MapOverview.kt

```diff
@@ -64,7 +64,8 @@ import com.google.maps.android.compose.rememberCameraPositionState
 fun MapOverview(
```

Comment:
Maybe we could update the comments for this composable according to the new parameters introduced